src/clib/xtg/grd3d_export_grdeclprop.c

```diff
@@ -33,7 +33,7 @@ grd3d_export_grdeclprop(int nx,
     int i, j;
     double min, max, avg;
     FILE *fc;
-    char uformat[33];
+    char * uformat;
```

Comment:
`const char * uformat;` ?

Reply:
Yes according to this, `const char *` is preferred: https://www.reddit.com/r/C_Programming/comments/bg3bdq/when_should_you_use_const_char/

Reply:
Yepp, done, did a quick force push on this one as it's such a small change